docs/book/src/developer/guide.md

```diff
@@ -147,6 +148,10 @@ To test another iteration, you'll need to follow the steps to build, push, updat
 
 **Additional ClusterAPI KubeCon talks**
 
+* [How Adobe Planned For Scale With Argo CD, Cluster API, And VCluster - October 2022](https://www.youtube.com/watch?v=p8BluR5WT5w)
+* [Bare-Metal Chronicles: Intertwinement Of Tinkerbell, Cluster API And GitOps - October 2022](https://www.youtube.com/watch?v=NCFUUjTw6hA)
+* [Running Isolated VirtualClusters With Kata & Cluster API - October 2022](https://www.youtube.com/watch?v=T6w3YrExorY)
+* [SIG Cluster Lifecycle Intro - October 2022](https://www.youtube.com/watch?v=0Zo0cWYU0fM)
```

Comment:
@sbueringer @richardcase what about adding also tutorials + links to tutorials web sites?

Reply:
Sounds good

In our case I would link:
* sched with video: https://kccncna2022.sched.com/event/1BZDs (also gives some context in the talk description)
* tutorial repo: https://github.com/ykakarap/kubecon-na-22-capi-lab

Reply:
The sched link also has a link to the youtube video so that is a plus.

Reply:
Yup that's why I wrote "sched with video" :D

Reply:
Added

Reply:
Let's wait a bit what Richard says, he had another separate tutorial (~ building a provider)

Reply:
Sorry for the delay, for our tutorial:

- https://kccncna2022.sched.com/event/182Ha/tutorial-so-you-want-to-develop-a-cluster-api-provider-anusha-hegde-nirmata-winnie-kwon-vmware-richard-case-suse-avishay-traeger-red-hat
- https://capi-samples.github.io/kubecon-na-2022-tutorial/

Reply:
Thank you Richard! Added.